db/schema.rb

```diff
@@ -69,12 +69,13 @@
   end
 
   create_table "bookmarks", force: :cascade do |t|
-    t.string "identifier"
-    t.datetime "date_value"
-    t.integer "id_value"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.index ["identifier"], name: "index_bookmarks_on_identifier", unique: true
+    t.integer "rank"
```

Comment:
Could we use a different word than "rank"? I think there's a desire to have an actual user ranking/rating system in the future, so this column name could be confusing when that happens. How about "order" or "position"/"pos"?